app/controllers/users/passwords_controller.rb

```diff
@@ -6,9 +6,14 @@ class Users::PasswordsController < Devise::PasswordsController
   layout "session"
 
   def create
-    super do |user|
-      if user.nil? || user.new_record?
-        flash.now[:notice] = ts("We couldn't find an account with that email address or username. Please try again?")
+    if User.find_for_authentication(login: resource_params[:login])&.protected_user
+      flash[:error] = t(".reset_blocked", contact_abuse_link: view_context.link_to(t(".contact_abuse"), new_abuse_report_path)).html_safe
+      redirect_to :root
```

Comment:
Can you return here instead of having an else block?

Reply:
Good idea! I'm also changing `:root` to `root_path` -- while a quick Google leaves me fairly certain symbols are treated as `_path` rather than `_url`, I'd rather be explicit so the next person to look at the code can be sure without Googling.